components/financial_assistance/app/controllers/financial_assistance/benefits_controller.rb

```diff
@@ -10,20 +10,26 @@ class BenefitsController < FinancialAssistance::ApplicationController
     before_action :set_cache_headers, only: [:index]
 
     def index
+      authorize @applicant, :index?
+
       save_faa_bookmark(request.original_url)
       set_admin_bookmark_url
       render layout: 'financial_assistance_nav'
       # @insurance_kinds = FinancialAssistance::Benefit::INSURANCE_TYPE
     end
 
     def new
+      authorize @applicant, :index?
```

Comment:
```suggestion
      authorize @applicant, :new?
```

@mdkaraman Let's use `:new?` instead of `:index?` for consistency and readability.

Reply:
Good catch!  Updating now.